### PR TITLE
Cleanup install cookbook from unused attributes and tests

### PR DESCRIPTION
**Pull request by awslitvit**
### Description of changes
Cleanup install cookbook from unused attributes and tests:
* [Cleanup unused attributes in install cookbooks](https://github.com/aws/aws-parallelcluster-cookbook/commit/eb897cb50e5313c66f09058d5581050609bd8111) 
* [Remove empty ChefSpec tests from install cookbook](https://github.com/aws/aws-parallelcluster-cookbook/commit/1b6b4368dd6d36d4d52199cd94a5b10ae309d38d)

### Tests
* `./kitchen.ec2.sh environment-install verify nfs -c 6`: OK
* `./kitchen.ec2.sh environment-config verify nfs -c 6`: OK

### Checklist
- [x] Make sure you are pointing to **the right branch** and add a label in the PR title (i.e. **2.x** vs **3.x**)
- [x] Check all commits' messages are clear, describing what and why vs how.
- [x] Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- [x] Check if documentation is impacted by this change.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cookbooks/aws-parallelcluster-install/attributes/default_ubuntu20.rb [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# Ubuntu 20 default attributes for aws-parallelcluster-install
-
-return unless platform?('ubuntu') && node['platform_version'] == "20.04"
-
-default['nfs']['service_provider']['idmap'] = Chef::Provider::Service::Systemd
```
Comment (enrico-usai) [inline cookbooks/aws-parallelcluster-install/attributes/default_ubuntu20.rb:5]: If I'm not wrong these attributes are defined here to override properties of the nfs third party cookbook.
Are we sure that is safe to remove them?

**Review by enrico-usai — APPROVED** [review]

File: cookbooks/aws-parallelcluster-install/attributes/default_ubuntu20.rb (hunk 1 above)
Comment (enrico-usai) [reply cookbooks/aws-parallelcluster-install/attributes/default_ubuntu20.rb:5]: These attributes have been added long time ago: https://github.com/aws/aws-parallelcluster-cookbook/commit/f2685b08dbc585fbf1ee5eca5e0be2bcc6227e8a#diff-030d34d25500e7467089d4e90fa949f3f27db2f89e1f47ba7a36de7c02e95748

Comment (awslitvit) [reply cookbooks/aws-parallelcluster-install/attributes/default_ubuntu20.rb:5]: I hope our tests (see description) are sufficient to certify we are safe to remove these attributes. If not, we will integrate them with missing checks.

**Comment by awslitvit** [comment]
Skipping system tests since they passed before the rebase.

**Pull request merged**